### ISSUE TITLE
Error when using LoRA on Colab

### ISSUE THREAD

**Issue by tbuyle** [lllyasviel/Fooocus#867]
**Describe the problem**
When I try to use a LoRA on Google Colab, I have the below error.

First try has the message "Fooocus has tried to move the corrupted file to /content/Fooocus/models/loras/my_loRa.safetensors.corrupted  You may try again now and Fooocus will download models again. "

Then if I try to generate a new image, i get the "AttributeError: 'NoneType' object has no attribute 'patcher'" error and the interface is stuck at "Initalizing"

I had the same problem with various LoRA downloaded from civitai.

**Full Console Log**
Traceback (most recent call last):
  File "/content/Fooocus/modules/patch.py", line 473, in loader
    result = original_loader(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/safetensors/torch.py", line 259, in load_file
    with safe_open(filename, framework="pt", device=device) as f:
safetensors_rust.SafetensorError: Error while deserializing header: HeaderTooLarge

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/content/Fooocus/modules/async_worker.py", line 642, in worker
    handler(task)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/content/Fooocus/modules/async_worker.py", line 282, in handler
    pipeline.refresh_everything(refiner_model_name=refiner_model_name, base_model_name=base_model_name, loras=loras)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/content/Fooocus/modules/default_pipeline.py", line 233, in refresh_everything
    refresh_loras(loras)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/content/Fooocus/modules/default_pipeline.py", line 154, in refresh_loras
    model = core.load_sd_lora(model, filename, strength_model=weight, strength_clip=weight)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/content/Fooocus/modules/core.py", line 81, in load_sd_lora
    lora = fcbh.utils.load_torch_file(lora_filename, safe_load=False)
  File "/content/Fooocus/backend/headless/fcbh/utils.py", line 13, in load_torch_file
    sd = safetensors.torch.load_file(ckpt, device=device.type)
  File "/content/Fooocus/modules/patch.py", line 489, in loader
    raise ValueError(exp)
ValueError: Error while deserializing header: HeaderTooLarge
File corrupted: /content/Fooocus/models/loras/xl_more_art-full_v1.safetensors 
Fooocus has tried to move the corrupted file to /content/Fooocus/models/loras/xl_more_art-full_v1.safetensors.corrupted 
You may try again now and Fooocus will download models again. 

Exception in thread Thread-2 (worker):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
Total time: 1.86 seconds
  File "/content/Fooocus/modules/async_worker.py", line 649, in worker
    pipeline.prepare_text_encoder(async_call=True)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/content/Fooocus/modules/default_pipeline.py", line 216, in prepare_text_encoder
    fcbh.model_management.load_models_gpu([final_clip.patcher, final_expansion.patcher])
AttributeError: 'NoneType' object has no attribute 'patcher'

**Comment by lllyasviel** [comment]
are you sure that the files are not corrupted?
if is that case, give the link to that lora

**Comment by tbuyle** [comment]
https://civitai.com/models/124347/xlmoreart-full-xlreal-enhancer?modelVersionId=152309

I had the same problem with these LoRA as well :
https://civitai.com/models/185166/tim-burton-art-style-sdxl
https://civitai.com/models/184896/dragonballartstyle

**Comment by rickrender** [comment]
I've run into the same issue but on Windows 11, when trying to use these models, oddly it works for this model though...the same models work in A1111 so not sure what's happening, hope it can be resolved soon:

Working: 
https://civitai.com/models/158999/doctor-diffusions-vintage-line-art-xl-lora

Not Working: 
https://civitai.com/models/185492?modelVersionId=208242 
https://civitai.com/models/186670?modelVersionId=209594 
-------
Traceback (most recent call last):
  File "C:\Users\retro\Fooocus\Fooocus\modules\patch.py", line 473, in loader
    result = original_loader(*args, **kwargs)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\safetensors\torch.py", line 259, in load_file
    with safe_open(filename, framework="pt", device=device) as f:
safetensors_rust.SafetensorError: Error while deserializing header: HeaderTooLarge

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\retro\Fooocus\Fooocus\modules\async_worker.py", line 645, in worker
    handler(task)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\Fooocus\modules\async_worker.py", line 285, in handler
    pipeline.refresh_everything(refiner_model_name=refiner_model_name, base_model_name=base_model_name, loras=loras)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\Fooocus\modules\default_pipeline.py", line 233, in refresh_everything
    refresh_loras(loras)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\Fooocus\modules\default_pipeline.py", line 154, in refresh_loras
    model = core.load_sd_lora(model, filename, strength_model=weight, strength_clip=weight)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\Fooocus\modules\core.py", line 81, in load_sd_lora
    lora = fcbh.utils.load_torch_file(lora_filename, safe_load=False)
  File "C:\Users\retro\Fooocus\Fooocus\backend\headless\fcbh\utils.py", line 13, in load_torch_file
    sd = safetensors.torch.load_file(ckpt, device=device.type)
  File "C:\Users\retro\Fooocus\Fooocus\modules\patch.py", line 489, in loader
    raise ValueError(exp)
ValueError: Error while deserializing header: HeaderTooLarge
File corrupted: C:\Users\retro\Fooocus\Fooocus\models\loras\MPeck.safetensors
Fooocus has tried to move the corrupted file to C:\Users\retro\Fooocus\Fooocus\models\loras\MPeck.safetensors.corrupted
You may try again now and Fooocus will download models again.

Exception in thread Thread-2 (worker):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "C:\Users\retro\Fooocus\Fooocus\modules\async_worker.py", line 652, in worker
    pipeline.prepare_text_encoder(async_call=True)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\python_embeded\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\retro\Fooocus\Fooocus\modules\default_pipeline.py", line 216, in prepare_text_encoder
    fcbh.model_management.load_models_gpu([final_clip.patcher, final_expansion.patcher])
AttributeError: 'NoneType' object has no attribute 'patcher'

**Comment by tbuyle** [comment]
I try using the Tim Burton's LoRA again - using the same file from my disk and uploading it to the Colab.

First test it worked - Then the next day it didn't.

I suspected there was something wrong with the file - So I downloaded the file from the Colab... And realize it was smaller than the original.

→ It seems the problem is actually on Colab that says a file import is completed when it is not. The .safetensor file is thus a partial upload - and a corrupted LoRA that can not be used.

I think this can thus be closed.
